docs/articles/advanced_topics/generic_host.md

```diff
@@ -0,0 +1,141 @@
+
+Next, it configures the services that the host will use by calling the `ConfigureServices()` method. In this case, it adds a new `YourBotHost` service, which is a class that you'll need to define next.
+
+Finally, it calls the `RunConsoleAsync()` method to start the host and begin running your service. That's it! With just a few lines of code, you can use the .NET Generic Host to run your application as a service.
+
+## Setting up your Service
+You'll need to create a class which implements the `IHostedService` interface, which defines two methods: `StartAsync()` and `StopAsync()`. These methods are called by the host when your application starts and stops, respectively.
+
+```cs
+public sealed class YourBotHost : IHostedService
+{
+    private readonly ILogger<YourBotHost> _logger;
+    private readonly IHostApplicationLifetime _applicationLifetime;
+    private readonly DiscordClient _discordClient;
+
+    public YourBotHost(ILogger<YourBotHost> logger, IHostApplicationLifetime applicationLifetime)
+    {
+        this._logger = logger;
+        this._applicationLifetime = applicationLifetime;
+        this._discordClient = new(new()
+        {
+            Token = "YourBotTokenHere",
+            TokenType = TokenType.Bot,
+            Intents = DiscordIntents.AllUnprivileged
+        });
+    }
+
+    public async Task StartAsync(CancellationToken token)
+    {
+        await _discordClient.ConnectAsync();
+        // Other startup things here
+    }
+
+    public async Task StopAsync(CancellationToken token)
+    {
+        await _discordClient.DisconnectAsync();
+        // More cleanup possibly here
+    }
+}
+```
+The `StartAsync()` method contains the code that runs when your application starts. In this case, the `DiscordClient` connects to the Discord API.
+
+The `StopAsync()` method contains the code that runs when your application is shut down. In the case of a bot, this might involve closing connections to external APIs, releasing resources, or performing other cleanup tasks.
+
+By implementing these methods, you can ensure that your application starts and stops cleanly, and that any necessary resources are properly managed. This makes it easier to build robust, reliable applications that can be run as services.
+
+With this class, you can easily create and run your own bot services using the .NET Generic Host. Just replace the Token property with your own Discord bot token, and you're ready to go!
+
+## Using Serilog with the Generic Host
+Logging is an important part of any application, especially one that runs as a service. Fortunately, the .NET Generic Host makes it easy to integrate with popular logging libraries, like Serilog.
+
+### Dependencies
+You will need the [`Serilog.Extensions.Hosting`](https://www.nuget.org/packages/Serilog.Extensions.Hosting) package (along with [`Serilog`](https://www.nuget.org/packages/Serilog) itself with whichever sinks you'd prefer), which is available from NuGet.
```

Comment:
`is` should be `are` ("[...] are available from NuGet.")

Reply:
in this vain, the expression `you would` (you'd) isn't exactly apt for the structure of the sentence